core/src/main/resources/schema/app/20210112140444_CB-10241-Data_Hub_page_should_show_the_runtime_version_instead_of_the_blueprint_version.sql

```diff
@@ -0,0 +1,11 @@
+-- // CB-10241-Data Hub page should show the runtime version instead of the blueprint version
+-- Migration SQL that makes the change goes here.
+ALTER TABLE stack DROP COLUMN IF EXISTS stackversion;
+ALTER TABLE stack ADD COLUMN stackversion VARCHAR (255);
```

Comment:
Can `stackversion` already exist with a different definition, or rather just want to make this migration idempotent? If the latter, `ADD COLUMN IF NOT EXISTS` would be better so that no `DROP COLUMN` is needed.

Reply:
You are right, corrected!